site/_filters/trim-html.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const clipper = require('text-clipper').default;
+
+/**
+ * Trims HTML string to 280 characters.
```

Comment:
Would this cause the article to abruptly end in the RSS reader?

Reply:
It seems to properly close out the html tags. here's an example:

```xml
<entry>
  <title>Style guide</title> 
  <link>https://developer.chrome.com/en/style-guide/</link> 
  <updated>2021-01-15T10:29:47-08:00</updated> 
  <id>https://developer.chrome.com/en/style-guide/</id> 
  <content type="html"><!DOCTYPE html> <html lang="en"> <head> <meta charset="UTF-8"> <meta name="viewport" content="width=device-width, initial-scale=1.0"> <title>Style guide - Chrome Developers</title> <link rel="canonical" href="https://developer.chrome.com/style-guide/"> <meta property="og:site_name" content="Chrome Developers"> <meta property="og:title" content="Style guide - Chrome Developers"> <meta property="og:type" content="website"> <meta property="og:url" content="https://developer.chrome.com/style-guide/"> <meta name="description" content="Build the next generation of web experiences."> <meta name="twitter:description" content="Build the next generation of web experiences."> <meta property="og:description" content="Build the next generation of web experiences."> <link rel="apple-touch-icon" sizes="180x180" href="https://developer.chrome.com/images/meta/apple-touch-icon.png"> <link rel="icon" type="image/png" sizes="32x32" href="https://developer.chrome.com/images/meta/favicon-32x32.png"> <link rel="icon" type="image/png" sizes="16x16" href="https://developer.chrome.com/images/meta/favicon-16x16.png"> <meta name="theme-color" content="#ffffff"> <link rel="preload" as="font" crossorigin="" href="https://developer.chrome.com/fonts/google-sans-v2003/regular/latin.woff2"> <link rel="preload" as="font" crossorigin="" href="https://developer.chrome.com/fonts/google-sans-v2003/medium/latin.woff2"> <link rel="stylesheet" href="https://developer.chrome.com/css/main.css"> <script> let ctaUrl = 'https://policies.google.com/technologies/cookies'; let savedCtaUrl = localStorage.getItem('user-cookies'); if …</script></head></html></content> 
  </entry>
```

Reply:
Sorry, I meant that it seems like it would cut the article off. Whereas css-tricks and dave seem to include the entire content of their article. Otherwise someone reading in an rss reader would only get part of the article and then it would stop.